linmod/tests/test_hhs_agg.py

```diff
@@ -0,0 +1,292 @@
+import polars as pl
+import pytest
+from polars.testing import assert_frame_equal
+
+from linmod.models import InfectionWeightedAggregator
+
+
+def test_infection_weighted_aggregator():
```

Comment:
these feel like a few different tests? you could put the common inputs to the tests (if there is such a thing) outside the testing function, and then break up into a few different testing functions

Reply:
Yes, splitting is possible. Now #65